src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -176,6 +179,24 @@
     </DynamicEnumProperty.Metadata>
   </DynamicEnumProperty>
 
+  <!-- TODO: Shortlink for help URL -->
+  <BoolProperty Name="UseApplicationFramework"
+                DisplayName="Application Framework"
+                Description="Enable the Visual Basic Application Framework for this project."
+                HelpUrl="https://docs.microsoft.com/visualstudio/ide/reference/application-page-project-designer-visual-basic#enable-application-framework"
+                Category="General">
```

Comment:
I would stick this in the `ApplicationFramework` category directly, to be more consistent with how properties behave elsewhere.

As it stands, checking this box toggles a bunch of properties which are likely not visible on screen at that moment. Generally when you check a box that enables other properties, they appear immediately after that checkbox (e.g. generate documentation, ...).